README.md

```diff
@@ -119,6 +119,44 @@ interact with the webapp.
 bin/dc-logs -f
 ```
 
+### Database
+
+The database, named `new_to_nl_development` would be accessible on the
+port `5345` in the local host.  By default, the username is `postgres`
+and the password is namesake.  You can explore the database using
```

Comment:
I'm unsure if the password is "postgres" or "namesake". Which did you intend?

Reply:
The password is "postgres".  I have to rephrase that.

Reply:
Cool, thank you!